fb_instant_articles.module

```diff
@@ -46,24 +116,24 @@ function fb_instant_articles_page_attachments_alter(array &$page) {
  * Implements hook_init().
  */
 function fb_instant_articles_init() {
-  $enable_transformer_logging = \Drupal::config('fb_instant_articles_enable_logging');
-  $appender = array(
+  $enable_transformer_logging = \Drupal::config('fb_instant_articles.base_settings')->get('enable_logging');
+  $appender = [
     'class' => $enable_transformer_logging ? '\Drupal\fb_instant_articles\DrupalLoggerAppender' : 'LoggerAppenderNull',
```

Comment:
LoggerAppenders are a FB SDK thing that provides a way to forward log messages from the SDK to elsewheres, like Drupal watchdog.  So I'm going to guess that `LoggerAppenderNull` is a keyword.